src/app/space/wizard/space-wizard.component.ts

```diff
@@ -120,16 +128,27 @@ export class SpaceWizardComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
-    const srumTemplates = this.spaceTemplates.filter(template => template.name === 'Scenario Driven Planning');
-    if (srumTemplates && srumTemplates.length > 0) {
-      this.selectedTemplate = srumTemplates[0];
-    }
     this.context.current.subscribe((ctx: Context) => {
       if (ctx.space) {
         this.currentSpace = ctx.space;
         console.log(`ForgeWizardComponent::The current space has been updated to ${this.currentSpace.attributes.name}`);
       }
     });
+    this.spcaeTemplateService.getSpaceTemplates()
+      .subscribe((templates: ProcessTemplate[]) => {
+        this.spaceTemplates = templates.filter(t => t.attributes['can-construct']);
+        this.selectedTemplate = !!this.spaceTemplates.length ? this.spaceTemplates[0] : null;
+      }, () => {
+        console.log('Error on getting space templates');
```

Comment:
console.log should be removed

Reply:
removed in 269bc6e6789687cadcf3042c7bb63345a6beef63